### DISCUSSION TITLE
Branding for the site and the podcast

### DISCUSSION THREAD

**Discussion by daveverwer** [SwiftPackageIndex/SwiftPackageIndex-Server#2152] [Ideas]
So I started working on #2137 and started thinking about branding for the podcast, and it turned into something else I’ve wanted to refresh for a while. The original logo was knocked up in a few minutes, and while it has lasted longer than I expected, it was never meant to be a long-term solution.

I always wanted to play with the idea of a literal swift “package” and put the Swift logo on a box, and so while I started playing with this for the podcast, I think we may have a draft of a new logo out of it. It’s not good, but if we wanted something good, then we’d get a designer to do it.

<img src="https://user-images.githubusercontent.com/5180/203574974-66838722-7ac0-4935-a55f-ab251e70f300.png" width="300"> <img src="https://user-images.githubusercontent.com/5180/203574983-e117c1ad-d3f0-4e1c-8ef4-af9aa65731a3.png" width="300"> 

Feedback is welcome, of course.

There’s also a colour version that I thought we could use for the podcast artwork, so it’s not just the site logo repeated. I have an idea for this using a colour version of this logo:

<img src="https://user-images.githubusercontent.com/5180/203574950-b8ae6071-fb8e-4400-b2d5-39be2b44397c.png" width="300"> <img src="https://user-images.githubusercontent.com/5180/203577232-04da8ec9-2d1b-44cd-b562-cfa4e76feba4.png" width="300">

That gradient is too strong as a background to it, but something along those lines.

**Comment by finestructure** [comment]
These look really nice!

One thing that I like about the current logo is that it is pretty unique in shape. It's quite recognisable, even in the context of Swift logos, which are often orange, have the bird, and sometimes even something rectangular or boxy going on.

I also like that it looks a bit like a badge, which in my mind goes well with being an index.

Maybe it's just a matter of assembling all our logo use cases and comparing the two side by side to see where we're at?
